Base/Python/slicer/util.py

```diff
@@ -2307,7 +2307,10 @@ def infoDisplay(text, windowTitle=None, parent=None, standardButtons=None, **kwa
   """Display popup with a info message.
 
   If there is no main window then the text is only logged (at info level).
+  Skipped during testing.
```

Comment:
Considering adding note like this:

```
.. note:: When testing mode is enabled, the confirmation dialog is skipped and a message indicating so is also logged.
```

Reply:
It might be simpler to say that if the application runs in testing mode then instead of showing the message in a popup, the message is just logged.